### PR TITLE
DOC-12189 cbq in tools package

### PR DESCRIPTION
**Pull request by ggray-cb**
This PR handles updating the CB Server docs for the addition of the cbq tool into the downloadable packages. I've also edited the page to meet our current documentation standards, and make later updating easier.

[Here's a preview of the changes](https://preview.docs-test.couchbase.com/cbq-in-tools-package/server/current/cli/cli-intro.html).

### CODE REVIEW

File: modules/cli/pages/cli-intro.adoc [hunk 1]
```diff
@@ -1,70 +1,82 @@
 = CLI Reference
-:description: Couchbase Server command-line interface (CLI) tools are provided to manage and monitor clusters, servers, vBuckets, XDCR, and so on.
+:description: The command-line interface (CLI) tools lets you manage and monitor your Couchbase Server installation including clusters, servers, vBuckets, and XDCR.
```
Comment (simon-dew) [inline modules/cli/pages/cli-intro.adoc]: subject-verb agreement

```suggestion
:description: The command-line interface (CLI) tools let you manage and monitor your Couchbase Server installation including clusters, servers, vBuckets, and XDCR.
```

File: modules/cli/pages/cli-intro.adoc [hunk 2]
```diff
@@ -1,70 +1,82 @@
...
+:tools-ver: 7.6.2
 
 [abstract]
 {description}
 
-By default, the CLI tools are installed into the following locations on each platform:
+The Couchbase Server installation process installs the command-line tools.
+The location of these tools depends on your platform:
```
Comment (simon-dew) [inline modules/cli/pages/cli-intro.adoc]: Maybe make it clear that this is where you can find the tools after installation?

```suggestion
When you have installed Couchbase Server, you can find the command-line tools in one of the following locations, depending on your your platform:
```

File: modules/cli/pages/cli-intro.adoc [hunk 3]
```diff
@@ -1,70 +1,82 @@
...
+* [.path]`/opt/couchbase/bin`
+* [.path]`/opt/couchbase/bin/install`
+* [.path]`/opt/couchbase/bin/tools`
 
 | Windows
-| [.path]_C:\Program Files\couchbase\server\bin_, [.path]_C:\Program Files\couchbase\server\bin\install_, and [.path]_C:\Program Files\couchbase\server\bin\tools_.
+a|
+* [.path]`C:\Program Files\couchbase\server\bin`
+* [.path]`C:\Program Files\couchbase\server\bin\install`
+* [.path]`C:\Program Files\couchbase\server\bin\tools`.
```
Comment (simon-dew) [inline modules/cli/pages/cli-intro.adoc]: Remove period for consistency

```suggestion
* [.path]`C:\Program Files\couchbase\server\bin\tools`
```

File: modules/cli/pages/cli-intro.adoc [hunk 4]
```diff
@@ -1,70 +1,82 @@
...
-[.path]_/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/tools_
-
-[.path]_/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/install_
+a| 
+* [.path]`/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin`
+* [.path]`/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/tools`
+* [.path]`/Applications/Couchbase Server.app/Contents/Resources/couchbase-core/bin/install`
 |===
 
 == Managing diagnostics
```
Comment (simon-dew) [inline modules/cli/pages/cli-intro.adoc]: Title case for consistency with other headings?

```suggestion
== Managing Diagnostics
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Nice use of attributes for the tools version. Some nitpicks

File: modules/cli/pages/cli-intro.adoc [hunk 5]
```diff
@@ -1,70 +1,82 @@
...
 [#version-compatibility]
 == Version Compatibility
 
-The versions of the utilities are the same as the Couchbase Server versions that the utilities are from.
+The versions of the utilities installed by the tools package are the same as the corresponding Couchbase Server installation package.
 
-The 7.6.0 `cbimport`, `cbexport`, and `cbbackupmgr` utilities can be run against the following Couchbase Server versions:
+The {tools-ver} `cbimport`, `cbexport`, `cbbackupmgr`, and `cbq` utilities are compatible with the following Couchbase Server versions:
 
 * 7.6.0
```
Comment (hyunjuV) [inline modules/cli/pages/cli-intro.adoc]: * 7.6.0, 7.6.1

**Review by hyunjuV — APPROVED** [review]
Just the one nit picky change:
On line 69, 81:
Instead of 
* 7.6.0
should be
* 7.6.0, 7.6.1

Otherwise looks good.  Thank you for making all the changes.  Will go ahead and approve.

**Review by hyunjuV — APPROVED** [review]

**Pull request merged**